src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/ResolveSdksTaskTests.cs

```diff
@@ -219,7 +219,10 @@ public void ResolveSdkTiming ()
 			Assert.AreEqual (task.ReferenceAssemblyPaths[0], Path.Combine (referencePath, "MonoAndroid"), $"ReferenceAssemblyPaths should be {Path.Combine (referencePath, "MonoAndroid")}.");
 			var expected = Path.Combine (Root);
 			Assert.AreEqual (task.MonoAndroidToolsPath, expected, $"MonoAndroidToolsPath should be {expected}");
-			expected = Path.Combine (Root, "Darwin" + Path.DirectorySeparatorChar);
```

Comment:
Wait, what? Why is it checking for a `Darwin` directory in the first place?

Related: this will fail on Linux.

It *should* be `$"{Root}/{HostOSName}"`, except on Windows where `HostOSName` is not set.

I suppose *a* problem is that there's no easy way to get "HostOSName" from the unit tests... :-/

Reply:
I changed the test to ignore the trailing directory name.

This should pass on all platforms now.